CHANGELOG.md

```diff
@@ -5,8 +5,9 @@
 - New Onboarding UI
 - Show new registration label for recent patients in home screen
 - Redesigned patient card in search results
-- Redesigned patient card in recent patients screen
 - New illustrations for location permission screen, overdue screen, reports screen
+- Redesigned patient card in recent patients screen
```

Comment:
This (reordering of the line items) seems to been a rebase conflict issue. Not necessary to fix, something to watch out for in future.

Reply:
noted